### PR TITLE
eeo.classin version 4.2.6.353

### PR DESCRIPTION
**Pull request by oxygen-dioxide-2**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/50288)

### CODE REVIEW

File: manifests/e/eeo/classin/4.2.6.353/eeo.classin.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: eeo.classin
```
Comment (KaranKad) [inline manifests/e/eeo/classin/4.2.6.353/eeo.classin.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: EEOEducation.Classin
```

File: manifests/e/eeo/classin/4.2.6.353/eeo.classin.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: eeo.classin
```
Comment (KaranKad) [inline manifests/e/eeo/classin/4.2.6.353/eeo.classin.installer.yaml:4]:
```suggestion
PackageIdentifier: EEOEducation.Classin
```

File: manifests/e/eeo/classin/4.2.6.353/eeo.classin.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: eeo.classin
```
Comment (KaranKad) [inline manifests/e/eeo/classin/4.2.6.353/eeo.classin.yaml:4]:
```suggestion
PackageIdentifier: EEOEducation.Classin
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You need to update the folders and file names as well.

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**